frontend/src/scenes/pipeline/pipelineLogic.tsx

```diff
@@ -29,7 +29,7 @@ export const humanFriendlyTabName = (tab: PipelineTabs): string => {
         case PipelineTabs.Destinations:
             return 'Destinations'
         case PipelineTabs.AppsManagement:
-            return 'Apps Management'
+            return 'Apps management'
```

Comment:
We use "Sentence case" as a brand, also applies to the UI (https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md#additional-suggestions)